src/legacy/legacy-default-xrp-client.ts

```diff
@@ -22,6 +26,8 @@ export class LegacyXRPClientErrorMessages {
 
   public static readonly xAddressRequired =
     'Please use the X-Address format. See: https://xrpaddress.info/.'
+
```

Comment:
Seems like an enum would make sense here as well (dealing with a list of constants)

Reply:
It probably would, but since this is legacy we're not putting any effort into refactoring it.